GiniVision/Classes/Core/Screens/Camera/CameraViewController.swift

```diff
@@ -323,6 +327,18 @@ extension CameraViewController {
         }
     }
     
+    fileprivate func configurePopupViewForUnsupportedQR(_ newQRCodePopup: QRCodeDetectedPopupView,
+                                                        dismissCompletion: @escaping () -> Void) {
+        newQRCodePopup.backgroundColor = UIColor.from(giniColor:giniConfiguration.unsupportedQrCodePopupBackgroundColor)
+        newQRCodePopup.qrText.textColor = UIColor.from(giniColor: giniConfiguration.unsupportedQrCodePopupTextColor)
+        newQRCodePopup.qrText.text = .localized(resource: CameraStrings.unsupportedQrCodeDetectedPopupMessage)
+        newQRCodePopup.proceedButton.setTitle("✕", for: .normal)
+        newQRCodePopup.proceedButton.setTitleColor(giniConfiguration.unsupportedQrCodePopupButtonColor, for: .normal)
```

Comment:
Also need to set the color for the highlighted state (same as the `QRCodeDetectedPopupView` does on line 73):
```suggestion
        newQRCodePopup.proceedButton.setTitleColor(giniConfiguration.unsupportedQrCodePopupButtonColor, for: .normal)
newQRCodePopup.proceedButton.setTitleColor(giniConfiguration.unsupportedQrCodePopupButtonColor.withAlphaComponent(0.5),                                                   for: .highlighted)
```